plugins/api/src/data_object_modify_info.cpp

```diff
@@ -143,6 +144,7 @@ auto plugin_factory(const std::string& _instance_name,
                         op,                              // Operation
                         "api_data_object_modify_info",   // Operation name
                         clearModDataObjMetaInp,          // Null clear function
+                        irods::clearOutStruct_noop,
```

Comment:
No, I agree. Many of the things such as this are because I'm afraid of the work existing on my machine.

I'm going to clean up all of these changes as this PR comes to a close.

Reply:
Done